core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -387,33 +409,19 @@ class WorkspaceService(protected val userInfo: UserInfo,
     //Notice that we're kicking off Futures to do the aborts concurrently, but we never collect their results!
     //This is because there's nothing we can do if Cromwell fails, so we might as well move on and let the
     //ExecutionContext run the futures whenever
```

Comment:
nit: move this comment down by the line where we actually call `executionServiceCluster.abort`